frontend/src/App.jsx

```diff
@@ -25,6 +27,8 @@ function App() {
 
   return (
     <>
+    <NavBar />
```

Comment:
Corregir indentancion, falta una tabulacion hacia la derecha.
```suggestion
      <NavBar />
```

Reply:
Te recomiendo que te instales un extension de auto correccion de codigo para Visual Code.
Estas extensiones le dan formato automatico al codigo, se hace solo.

Yo tengo esta:
 - https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Un videito: https://www.youtube.com/watch?v=OmwrOgUXWBs&ab_channel=divcode